tests/webapp/api/test_auth.py

```diff
@@ -104,27 +104,6 @@ def test_post_no_auth():
 
 # TC Auth Login and Logout Tests
 
-def test_ldap_login_and_logout(test_user, webapp, monkeypatch):
```

Comment:
This is actually still a valid test.  But it needs to be changed a bit to work.  What we need is a new fixture that creates an LDAP user, rather than the ``test_user`` one that's used here.  Add that to the ``conftest.py`` file next to ``test_user``.  Something like:

    @pytest.fixture
    def test_ldap_user(request, transactional_db):
        user = User.objects.create(username="mozilla-ldap/user@foo.com",
                                   email='user@foo.com',
                                   is_staff=False)
    
        def fin():
            user.delete()
        request.addfinalizer(fin)
    
        return user